python/flink_agents/runtime/local_execution_environment.py

```diff
@@ -17,50 +17,92 @@
 #################################################################################
 from typing import Any, Dict, List
 
-from flink_agents.api.execution_enviroment import AgentsExecutionEnvironment
+from pyflink.common import TypeInformation
+from pyflink.datastream import DataStream, KeySelector
+from pyflink.table import Schema, StreamTableEnvironment, Table
+
+from flink_agents.api.execution_environment import (
+    AgentBuilder,
+    AgentsExecutionEnvironment,
+)
 from flink_agents.api.workflow import Workflow
 from flink_agents.runtime.local_runner import LocalRunner
 
 
-class LocalExecutionEnvironment(AgentsExecutionEnvironment):
-    """Implementation of AgentsExecutionEnvironment for local execution environment."""
+class LocalAgentBuilder(AgentBuilder):
+    """LocalAgentBuilder for building agent instance."""
 
+    __env: "LocalExecutionEnvironment"
     __input: List[Dict[str, Any]]
     __output: List[Any]
     __runner: LocalRunner = None
     __executed: bool = False
 
-    def __init__(self) -> None:
+    def __init__(
+        self, env: "LocalExecutionEnvironment", input: List[Dict[str, Any]]
+    ) -> None:
         """Init empty output list."""
-        self.__output = []
-
-    def from_list(self, input: list) -> 'AgentsExecutionEnvironment':
-        """Set input list of execution environment."""
+        self.__env = env
         self.__input = input
-        return self
+        self.__output = []
 
-    def apply(self, workflow: Workflow) -> 'AgentsExecutionEnvironment':
+    def apply(self, workflow: Workflow) -> AgentBuilder:
         """Create local runner to execute given workflow.
 
         Doesn't support apply multiple workflows.
         """
         if self.__runner is not None:
-            err_msg = "LocalExecutionEnvironment doesn't support apply multiple workflows."
+            err_msg = "LocalAgentBuilder doesn't support apply multiple workflows."
```

Comment:
Do we support multiple agents in the same job? If yes, `env.set_agent` doesn't make sense. If not, we should check in the env, not here.